ql/src/test/results/clientpositive/perf/tpcds30tb/tez/cbo_query15.q.out

```diff
@@ -15,6 +15,6 @@ HiveSortLimit(sort0=[$0], dir0=[ASC], fetch=[100])
               HiveFilter(condition=[AND(IS NOT NULL($2), IS NOT NULL($33))])
                 HiveTableScan(table=[[default, catalog_sales]], table:alias=[catalog_sales])
             HiveProject(d_date_sk=[$0])
-              HiveFilter(condition=[AND(=($6, 2000), =($10, 2))])
+              HiveFilter(condition=[AND(=($10, 2), =($6, 2000))])
```

Comment:
The order in the filter conditions is determined by `HiveFilterSortPredicates`. This change and other similar ones could happen due to change in selectivity estimates for each of the conjuncts. We should double check that these changes are expected. ^ @zabetak fyi

Reply:
Thanks for the info @jcamachor . @soumyakanti3578 also expressed some concerns around these changes. Truth is that it didn't cross my mind that condition order matters. I was suspecting that the changes were related to https://issues.apache.org/jira/browse/CALCITE-4073 or https://issues.apache.org/jira/browse/CALCITE-2450. We will check it out ;)